src/components/PostList/PostList.tsx

```diff
@@ -1,5 +1,32 @@
 import React from 'react';
 
-export const PostList: React.FC = () => (
-  <>Put the list here</>
+// Data
+import commentsFromServer from '../../api/comments';
+
+// Types
+import { Post } from '../../Types/post';
+import { Comment } from '../../Types/comment';
+
+import { PostInfo } from '../PostInfo';
+
+function getCommit(postId: number): Comment[] {
+  return commentsFromServer.filter(comment => comment.postId === postId);
+}
+
+type Props = {
+  posts: Post[],
+};
+
+export const PostList: React.FC<Props> = ({ posts }) => (
+  <ul className="PostList">
+    {posts.map((post: Post) => (
```

Comment:
No need to specify 'Post' as a type in map function since 'posts' is already typed as an array of 'Post'.

Reply:
isn't fixed